python_ta/debug/accumulation_table.py

```diff
@@ -137,12 +143,15 @@ def _setup_table(self) -> None:
         )[1].frame
         self._loop_lineno = inspect.getlineno(func_frame) + 1
 
-        for_node = get_for_node(func_frame)
-        if isinstance(for_node.target, astroid.Tuple):
-            self.loop_variables = {loop_var.name: [] for loop_var in for_node.target.elts}
-        else:
-            self.loop_variables[for_node.target.name] = []
+        node = get_loop_node(func_frame)
+        if isinstance(node, astroid.For) and isinstance(node.target, astroid.Tuple):
+            self.loop_variables = {loop_var.name: [] for loop_var in node.target.elts}
+        elif isinstance(node, astroid.For):
+            self.loop_variables[node.target.name] = []
 
+        assert self.loop_accumulators != {} or self.loop_variables != {}, (
+            "The loop accumulator and loop variables " "cannot be both empty "
```

Comment:
This string is formatted oddly (maybe due to running `black`). You can write the full message in just one string literal.